src/store/plugins/stop-updating-shortlink-when-menu-closed.plugin.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Redo the search results on lang change if the search query is defined
+ *
+ * @param {Vuex.Store} store
+ */
+const stopUpdatingShortLinkWhenMenuClosedPlugin = (store) => {
+    store.subscribe((mutation, state) => {
+        // when the user closes the menu, even though the share section is visible
+        // we shouldn't keep updating the short link (as the user has no intention to copy it right now, so we
+        // will just create unused short links if we do so)
+        // as soon as the menu is opened again (and the share section is still visible) we reactivate the update of the short link
+        if (mutation.type === 'setMenuDesktopOpen') {
```

Comment:
Wouldn't it be easier to create an "action" for `setMenuDesktopOpen` / `setShowMenuTray` and dispatch `toggleKeepUpdatingShortLink` from there?

Reply:
I'm in the process of replacing all the logic with one flag for the menu and the UI mode. I'm struggling with the tablet/desktop version to make it work properly...